examples/disable_keys.pp

```diff
@@ -0,0 +1,5 @@
+#Note: This is generally a bad idea. You should not disable keys.
```

Comment:
Not disabling keys, disabling verifying repository signatures / allowing installation of packages from unsigned / unverified sources.